cdash/CTestTestfile.cmake

```diff
@@ -0,0 +1,6 @@
+add_test(tests/ "make" "-C" "./tests/" "test" "TESTOPTS=++local")
```

Comment:
Do we want `++local` here?

Reply:
Not sure. How are the other autobuilds executed?

Reply:
It depends on the build. `++local` on netlrts and verbs uses forking on the head node instead of connecting to compute nodes. On most other architectures I think we have made it a no-op. It is specific to `make test` but not to `tests/`.

Unless a build specifically needs `++local` I would say we should not use it. Same goes for other uses in this PR.